### PR TITLE
create_object_no_files: fix modal selector.

### PR DESCRIPTION
**Pull request by lwrubel**
## Why was this change made?
create_object_no_files test was broken. Runs on staging now.

### CODE REVIEW

File: spec/features/create_object_no_files_spec.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
     # open a new version
     click_link 'Open for modification'
-    within '.modal-dialog' do
+    within '.modal-body' do
```
Comment (jcoyne) [inline spec/features/create_object_no_files_spec.rb:48]: I'm pretty sure the class we're looking for is `modal-dialog`. See https://github.com/sul-dlss/argo/blob/c3fef087978e09fe80770ce3af66538a505e704a/app/views/catalog/show.html.erb#L4

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/features/create_object_no_files_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/features/create_object_no_files_spec.rb:48]: The code on argo staging is currently in flux and there is a bug in the code. The test was looking correctly for modal-dialog.

Comment (lwrubel) [reply spec/features/create_object_no_files_spec.rb:48]: Got it. Should I just close this PR for now?

Comment (jcoyne) [reply spec/features/create_object_no_files_spec.rb:48]: Yes, I think so

**Pull request closed without merging**